packages/data-stores/src/starter-designs-queries/use-starter-designs-query.ts

```diff
@@ -95,10 +95,6 @@ function apiStarterDesignsToDesign( design: StarterDesign ): Design {
 
 	const is_externally_managed = design.theme_type === 'managed-external';
 
-	const is_bundled_with_woo_commerce = ( design.software_sets || [] ).some(
-		( { slug } ) => slug === 'woo-on-plans'
-	);
```

Comment:
Hello, @renatho! I have a question 🙋‍♀️ 
The original function `is_bundled_with_woo_commerce` checks for the presence of `woo-on-plans`. However, the PR modifies this to check if `software_sets.length > 0`. 
As far as checking `useBundleSettings`, there will be additional types of `software_sets`. In that case, these changes will break the original behavior in the future, am I correct?